### PR TITLE
Fix/inertia in cms

### PR DESCRIPTION
**Pull request by jschleicher**
move interial tensors into center-of-mass system
    
* specify center of mass for the links
* reduce inertial tensor accordingly
* unrelated: fix rviz config of test_urdf.launch

### CODE REVIEW

File: prbt_support/launch/robotmodel.rviz [hunk 1]
```diff
@@ -39,32 +39,16 @@ Visualization Manager:
...
-          Show Axes: false
-          Show Trail: false
-          Value: true
-        prbt_gripper_finger_left_link_bracket:
+        prbt_base:
           Alpha: 1
           Show Axes: false
           Show Trail: false
-          Value: true
-        prbt_gripper_finger_right_link:
```
Comment (hslusarek) [inline prbt_support/launch/robotmodel.rviz:57]: Why do we drop them? Because they are now part of a different package and have no relevance here?

**Comment by hslusarek** [comment]
"unrelated: fix rviz config of test_urdf.launch":
Maybe for clarity we should separate things which are not related to a PullRequest in the future.

File: prbt_support/urdf/prbt_macro.xacro [hunk 2]
```diff
@@ -229,7 +233,8 @@ limitations under the License.
     <link name="${prefix}link_5">
```
Comment (hslusarek) [inline prbt_support/urdf/prbt_macro.xacro:233]: What about the flange? It is still in the representation in which we state the inertia with regard to the link frame. Is this different representation ok?

**Review by hslusarek — CHANGES_REQUESTED** [review]
See comments.

File: prbt_support/launch/robotmodel.rviz (hunk 1 above)
Comment (jschleicher) [reply prbt_support/launch/robotmodel.rviz:57]: exactly. Trying to launch `test_urdf.launch` from kinetic-devel failed.

**Comment by jschleicher** [comment]
I've set `origin` of the flange to zero explicitly now and rebased onto kinetic-devel.
@hslusarek please merge before the next binary release. Thanks.

**Review by hslusarek — APPROVED** [review]

**Pull request merged**